packages/admin-stories/src/admin/form/RangeInput.tsx

```diff
@@ -37,24 +38,109 @@ const Thumb = styled.div`
-                        sliderProps={{ ThumbComponent: Thumb }}
-                    />
-                )}
-            />
-        </div>
+        <>
+            <h1>Final Form Range Input</h1>
+            <h3>with initialValues</h3>
+            <div style={{ width: "400px", paddingBottom: "30px" }}>
+                <Form
+                    onSubmit={(values) => {
+                        // values
+                    }}
+                    initialValues={{ price: { min: 0, max: 100 } }}
+                    render={({ handleSubmit, values, form, initialValues }) => (
+                        <>
+                            <Field
+                                component={FinalFormRangeInput}
+                                name="price"
+                                min={0}
+                                max={100}
+                                endAdornment={<span>€</span>}
+                                sliderProps={{ ThumbComponent: Thumb }}
+                            />
+                            <Button
+                                variant="contained"
+                                color="primary"
+                                onClick={() => {
+                                    for (const name in values) {
+                                        //required because reset removes values
+                                        form.change(name, (initialValues as Record<string, any>)[name]);
+                                    }
+                                }}
+                            >
+                                Reset
+                            </Button>
+                        </>
+                    )}
+                />
+            </div>
+            <div style={{ width: "400px", borderBottom: "1px solid grey" }} />
+            <h3>without initialValues</h3>
+            <div style={{ width: "400px", paddingBottom: "30px" }}>
+                <Form
+                    onSubmit={(values) => {
+                        // values
+                    }}
+                    render={({ values, form }) => (
+                        <>
+                            <Field
+                                component={FinalFormRangeInput}
+                                name="price"
+                                min={0}
+                                max={150}
+                                endAdornment={<span>€</span>}
+                                sliderProps={{ ThumbComponent: Thumb }}
+                            />
+                            <Button
+                                variant="contained"
+                                color="primary"
+                                onClick={() => {
+                                    for (const name in values) {
+                                        //required because reset removes values
+                                        form.change(name, { min: undefined, max: undefined });
```

Comment:
reset should be supported - what is the problem with (completely) removed value?

Reply:
thats true :) forgot about that :) i changed it to form.reset() :)